flink-filesystems/flink-s3-fs-presto/src/test/java/org/apache/flink/fs/s3presto/PrestoS3RecoverableWriterTest.java

```diff
@@ -87,7 +79,8 @@ public static void cleanUp() throws IOException {
 
 	@Test(expected = UnsupportedOperationException.class)
 	public void requestingRecoverableWriterShouldThroughException() throws Exception {
-		FlinkS3FileSystem fileSystem = (FlinkS3FileSystem) FileSystem.get(basePath.toUri());
+		URI s3Uri = URI.create(S3TestCredentials.getTestBucketUri());
```

Comment:
This is not using `TEST_DATA_DIR` anymore.

Reply:
It never used the `TEST_DATA_DIR` - the `getFileSystem()` only uses scheme and authority, path does not matter.